platform/platform-paxweb-jettyconfig/src/main/resources/jetty.xml

```diff
@@ -33,32 +33,44 @@
 <!DOCTYPE Configure PUBLIC "-//Mort Bay Consulting//
 DTD Configure//EN" "http://jetty.mortbay.org/configure_9_0.dtd">
 
-<Configure class="org.eclipse.jetty.server.Server">
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
 
     <Call name="setSessionIdManager">
         <Arg>
             <New class="org.codice.ddf.security.session.AttributeSharingHashSessionIdManager">
+                <Arg><Ref refid="Server" /></Arg>
             </New>
         </Arg>
     </Call>
+    <Call name="addBean">
+        <Arg>
+            <New id="SessionDataStoreFactory"
+              class="org.codice.ddf.security.session.AttributeSharingSessionDataStoreFactory"/>
+        </Arg>
+    </Call>
 
-    <Get name="handler">
-            <Set name="handlers">
-                <Array type="org.eclipse.jetty.server.Handler">
-                    <Item>
-                        <New id="RequestLog"
-                             class="org.eclipse.jetty.server.handler.RequestLogHandler"/>
-                    </Item>
-                </Array>
-            </Set>
-    </Get>
-    <Ref refid="RequestLog">
-        <Set name="requestLog">
-            <New id="RequestLogImpl" class="org.codice.ddf.pax.web.jetty.AccessRequestLog">
-                <Set name="resource">/access.xml</Set>
-                <Set name="quiet">true</Set>
-            </New>
-        </Set>
-    </Ref>
+    <Call name="addBean">
+        <Arg>
+            <New id="SessionCacheFactory"
+              class="org.eclipse.jetty.server.session.NullSessionCacheFactory"/>
+        </Arg>
+    </Call>
+
+    <!--<Get name="handler">-->
```

Comment:
Remove comment?

Reply:
I actually need to uncomment these. It was initially a temporary measure. Assuming @stustison still thinks it's valuable. I might add a mention in the documentation for how to use this `RequestLogHandler` when troubleshooting because I had never heard of it.

Reply:
I decided to put it back in. It was broken, even before the karaf upgrade, but I fixed it up and verified it works.